tests/cdc-endpos-between-transaction/000000010000000000000002.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN; -- {"xid":491,"lsn":"0/22E8318","timestamp":"2022-12-08 14:45:53.226340+0000"}
+INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1000, 'Fantasy', '2022-12-08 00:00:01+00');
+INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1001, 'History', '2022-12-09 00:00:01+00');
+INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1002, 'Adventure', '2022-12-10 00:00:01+00');
+INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1003, 'Musical', '2022-12-11 00:00:01+00');
+INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1004, 'Western', '2022-12-12 00:00:01+00');
+COMMIT; -- {"xid":491,"lsn":"0/22E8618","timestamp":"2022-12-08 14:45:53.226340+0000"}
+BEGIN; -- {"xid":492,"lsn":"0/22E8618","timestamp":"2022-12-08 14:45:53.226746+0000"}
+INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1005, 'Mystery', '2022-12-13 00:00:01+00');
+INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1006, 'Historical drama', '2022-12-14 00:00:01+00');
+INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1008, 'Thriller', '2022-12-15 00:00:01+00');
+COMMIT; -- {"xid":492,"lsn":"0/0","timestamp":"2022-12-08 14:45:53.226746+0000"}
```

Comment:
`"lsn":"0/0"`

This is strange. On every run, it's coming like this. I was expecting that Apply process will never finish but due to another behavior (i.e. reading the same file in the loop because SQL with `endpos` will never be there).

```log
14:50:32 118 INFO   ld_apply.c:78: Catching up from LSN 0/22E2F68 in "/var/lib/postgres/.local/share/pgcopydb/000000010000000000000002.sql"
14:50:32 118 INFO   ld_apply.c:84: Stopping at endpos LSN 0/22E9088
14:50:32 118 INFO   ld_apply.c:310: Replaying changes from file "/var/lib/postgres/.local/share/pgcopydb/000000010000000000000002.sql"
14:50:32 118 DEBUG  ld_apply.c:312: Read 12 lines in file "/var/lib/postgres/.local/share/pgcopydb/000000010000000000000002.sql"
14:50:32 118 DEBUG  ld_apply.c:362: BEGIN 491 LSN 0/22E8C60 @2022-12-08 14:50:29.918687+0000, previous LSN 0/22E2F68
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] BEGIN;
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] select pg_replication_origin_xact_setup($1, $2);
14:50:32 118 DEBUG  pgsql.c:1215: '0/22E8C60', '2022-12-08 14:50:29.918687+0000'
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1000, 'Fantasy', '2022-12-08 00:00:01+00');
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1001, 'History', '2022-12-09 00:00:01+00');
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1002, 'Adventure', '2022-12-10 00:00:01+00');
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1003, 'Musical', '2022-12-11 00:00:01+00');
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1004, 'Western', '2022-12-12 00:00:01+00');
14:50:32 118 DEBUG  ld_apply.c:429: COMMIT 491 LSN 0/22E8F60
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] COMMIT;
14:50:32 118 DEBUG  ld_apply.c:362: BEGIN 492 LSN 0/22E8F60 @2022-12-08 14:50:29.919467+0000, previous LSN 0/22E8F60
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] BEGIN;
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] select pg_replication_origin_xact_setup($1, $2);
14:50:32 118 DEBUG  pgsql.c:1215: '0/22E8F60', '2022-12-08 14:50:29.919467+0000'
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1005, 'Mystery', '2022-12-13 00:00:01+00');
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1006, 'Historical drama', '2022-12-14 00:00:01+00');
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] INSERT INTO "public"."category" ("category_id", "name", "last_update") VALUES (1008, 'Thriller', '2022-12-15 00:00:01+00');
14:50:32 118 DEBUG  ld_apply.c:429: COMMIT 492 LSN 0/0
14:50:32 118 NOTICE pgsql.c:1183: [TARGET] COMMIT;
14:50:32 118 DEBUG  pgsql.c:496: Connecting to [source] "postgres://postgres@source:/postgres?password=****"
14:50:32 118 NOTICE pgsql.c:1183: [SOURCE] update pgcopydb.sentinel set replay_lsn = $1 returning startpos, endpos, apply, write_lsn, flush_lsn, replay_lsn;
14:50:32 118 DEBUG  pgsql.c:1215: '0/0'
14:50:32 118 DEBUG  pgsql.c:415: Disconnecting from [source] "postgres://postgres@source:/postgres?password=****"
14:50:32 118 DEBUG  ld_apply.c:728: computeSQLFileName: 0/0 "/var/lib/postgres/.local/share/pgcopydb/000000010000000000000000.sql"
14:50:32 118 DEBUG  ld_apply.c:111: File "/var/lib/postgres/.local/share/pgcopydb/000000010000000000000000.sql" does not exists yet, retrying in 10000ms
14:50:37 118 DEBUG  pgsql.c:415: Disconnecting from [target] "postgres://postgres@target:/postgres?password=****"
```